stdlib/int32.gr

```diff
@@ -34,6 +67,13 @@ export let rec decr = (n: Int32) => {
   ret
 }
 
+/**
+ * Integer addition.
```

Comment:
I think it'd be better to have the doc comments be more descriptive about what a function _does_, rather than what it _is_. (@ospencer or @phated may want to revise the wording, idk)
```suggestion
 * Adds the two given numbers.
```